services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/UserContact.kt

```diff
@@ -48,12 +44,8 @@ data class ApplicationUserContact(
 fun ApplicationData.typedContacts(omit: String? = null): Set<ApplicationUserContact> =
     when (this) {
         is CableReportApplicationData ->
-            listOfNotNull(
-                    customerWithContacts.typedAs(HAKIJA),
-                    contractorWithContacts.typedAs(TYON_SUORITTAJA),
-                    representativeWithContacts?.typedAs(ASIANHOITAJA),
-                    propertyDeveloperWithContacts?.typedAs(RAKENNUTTAJA)
-                )
+            customersByRole()
+                .map { (role, customer) -> customer.typedAs(role) }
```

Comment:
typedAs could ne named to contactsTypedAs.

Reply:
Renamed as suggested.